x/cdp/client/rest/tx.go

```diff
@@ -24,17 +25,27 @@ func registerTxRoutes(cliCtx context.CLIContext, r *mux.Router) {
 
 func postCdpHandlerFn(cliCtx context.CLIContext) http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
-		// Decode PUT request body
 		var requestBody PostCdpReq
 		if !rest.ReadRESTReq(w, r, cliCtx.Codec, &requestBody) {
 			return
 		}
+
 		requestBody.BaseReq = requestBody.BaseReq.Sanitize()
 		if !requestBody.BaseReq.ValidateBasic(w) {
 			return
 		}
 
-		// Create and return msg
+		fromAddr, err := sdk.AccAddressFromBech32(requestBody.BaseReq.From)
```

Comment:
I've added from == sender/owner validation to all these requests. Let me know if this is invalid.